data_steward/resource_files/aou_vocab/concept.csv

```diff
@@ -10,7 +10,7 @@ concept_id	concept_name	domain_id	vocabulary_id	concept_class_id	standard_concep
 2000000008	More than one population	Observation	AoU_General	Answer	S	WhatRaceEthnicity_GeneralizedMultPopulations	2017-05-17	2099-12-31	
 2000000009	Not male, not female, prefer not to answer, or skipped	Observation	AoU_General	Answer	S	SexAtBirth_GeneralizedDiffSex	2017-05-17	2099-12-31	
 2000000010	Response removed due to invalid value	Observation	AoU_General	Answer	S	AoUDRC_ResponseRemoval	2017-05-17	2099-12-31	
-2000000011	State Information Suppressed for Privacy	Observation	AoU_General	Answer	S	AoUDRC_GeneralizedState	2017-05-17	2099-12-31
-2000000013	11 or more household members	Observation	AoU_General Answer	S	HouseSize_11+	2017-05-17	2099-12-31
+2000000011	State Information Suppressed for Privacy	Observation	AoU_General	Answer	S	AoUDRC_GeneralizedState	2017-05-17	2099-12-31	
+2000000013	11 or more household members	Observation	AoU_General	Answer	S	HouseSize_11+	2017-05-17	2099-12-31
```

Comment:
trailing tab is missing from this line
<img width="1115" alt="Screen Shot 2020-09-08 at 12 46 07 PM" src="https://user-images.githubusercontent.com/23404801/92505034-77e24d80-f1d1-11ea-9eb6-5bfc0bc63f55.png">